js/src/makegrid.js

```diff
@@ -232,20 +233,79 @@ var makeGrid = function (t, enableResize, enableReorder, enableVisib, enableGrid
             $(document.body).css('cursor', 'inherit').noSelect(false);
         },
 
+        /**
+         * Get data from localstorage or Cookies
+         *
+         * @param key the key to get data from
+         * @return {number} the data corresponding to the saved key
+         */
+        getStorageData: function (key) {
+            var data;
+            if (isStorageSupported('localStorage')) {
+                data = window.localStorage.getItem(key);
+            } else {
+                data = Cookies.get(key);
+            }
+            return data;
```

Comment:
```suggestion
            if (isStorageSupported('localStorage')) {
                return window.localStorage.getItem(key);
            }
            return Cookies.get(key);
```

A good way to avoid useless variables and useless else clauses